vyper/semantics/analysis/local.py

```diff
@@ -577,10 +589,10 @@ def visit_Return(self, node):
         self.expr_visitor.visit(node.value, self.func.return_type)
 
 
-class _ExprVisitor(VyperNodeVisitorBase):
+class ExprVisitor(VyperNodeVisitorBase):
```

Comment:
Removed the underscore because this needs to be imported into module validation in order to annotate the expressions of constant variables' values.

Reply:
hmm interesting, sounds like an import cycle